src/python/phenix_apps/apps/scorch/kafka/kafka.py

```diff
@@ -0,0 +1,167 @@
+import json, itertools, threading, time, sys, csv, re, os
+
+from phenix_apps.apps.scorch import ComponentBase
+from phenix_apps.common import logger, utils
+from kafka import KafkaConsumer
+from kafka.errors import KafkaError
+from datetime import datetime
```

Comment:
Python libraries, first-party libraries (phenix_apps) and third-party libraries should be defined in separate import groups